pkgs/tools/networking/pathvector/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "pathvector";
+  version = "5.11.1";
+
+  src = fetchFromGitHub {
+    owner = "natesales";
+    repo = "pathvector";
+    rev = "v${version}";
+    sha256 = "sha256-OgIDk+05bG2KrBQOyyMPaH0OJXU3gLM9OBab9lI+yXw=";
+  };
+
+  vendorSha256 = "sha256-R3o1L34FXbtRzJ1I2Xj4iWsiFJJWexGWYv2TmvhINe0=";
+
+  excludedPackages = [ "tests" ];
+
+  CGO_ENABLED = 0;
+
+  ldflags = [ "-X main.version=${version}" "-X main.commit=${src.rev}" "-X main.date=unknown" ];
+
+  checkPhase = ''
+    runHook preCheck
```

Comment:
What is this? If there are tests we want to run them and if the package has none nothing shoud break.

Reply:
Some of the tests require network access which causes them to fail.  I'm not sure what the proper way to approach this would be, but this works by bypassing the tests.  Definitely not ideal.

Reply:
``doCheck = false;`` disables tests. Go tests can be dissabled by renaming the functions to begin with `Skip`.

Reply:
I switched to using `doCheck = false` to disable the tests.  It seems that most of the tests rely on external services and there is no way to easily disable them without either patching or PRing upstream a way to skip those "external integration" tests.

If you rather I take a different approach to this, please let me know.